cassandane/tiny-tests/JMAPContacts/contact_get_emails_v4

```diff
@@ -0,0 +1,68 @@
+#!perl
+use Cassandane::Tiny;
+use Test::Deep ':v1';
+
+sub test_contact_get_emails_v4
+    :JMAPExtensions
+    ($self)
+{
+    my $jmap = $self->{jmap};
+
+    my $service = $self->{instance}->get_service("http");
+    $ENV{DEBUGDAV} = 1;
```

Comment:
I'm pretty sure there is a `$self->{carddav}` on the test case instance

Reply:
Yeah.  This test was copied from another and I didn't even pay attention to that.  Fixed.